### ISSUE TITLE
capsule-B F1 85.8?

### ISSUE THREAD

**Issue by StevenLOL** [andyweizhao/capsule_text_classification#1]
In the paper, the capsule-B F1 score on Reuters-Multilabel data set is 85.8, but the best score I can get is 83.7

```
python ./main.py -- model_type capsule-A --learning_rate 0.001
```

Epoch: 2  Val accuracy: 89.9%  Loss: 0.0612
ER: 0.095 Precision: 0.635 Recall: 0.575 F1: 0.566
Epoch: 3  Val accuracy: 93.3%  Loss: 0.0391
ER: 0.594 Precision: 0.912 Recall: 0.770 F1: 0.816
Epoch: 4  Val accuracy: 94.7%  Loss: 0.0326
**ER: 0.615 Precision: 0.939 Recall: 0.788 F1: 0.837**
Epoch: 5  Val accuracy: 95.8%  Loss: 0.0299
ER: 0.428 Precision: 0.948 Recall: 0.692 F1: 0.777
Epoch: 6  Val accuracy: 96.0%  Loss: 0.0272
ER: 0.348 Precision: 0.958 Recall: 0.661 F1: 0.759

**Comment by andyweizhao** [comment]
Thank you for your interest. This repository hosts the experimental and clean code with all the main function. The next update will include more details.
